config.toml.example

```diff
@@ -262,6 +262,10 @@
 # standard library.
 #debuginfo-only-std = false
 
+# Enable debuginfo for the extended tools: cargo, rls, rustfmt
+# Adding debuginfo increases their sizes by a factor of 3-4.
```

Comment:
I realize that these numbers are just copied from old comments, but could you verify that building Cargo (for example) with debuginfo is a 3-4x increase? I'd prefer not to perpetuate potentially wrong information into more places.

Reply:
Hmm, actually, I get 15MB without debuginfo, 95MB with!  And that's "without" that actually still includes some debuginfo from linked std etc. -- the fully stripped size is 9.2MB.

Let's avoid numbers though -- how about I'll just say that it's several times larger.